bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/aQGC_VVjj_hadronic/aQGC_WMhadWMhadJJ_EWK_LO_NPle1/aQGC_WMhadWMhadJJ_EWK_LO_NPle1_proc_card.dat

```diff
@@ -0,0 +1,8 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set complex_mass_scheme False
+import model SM_LS_LM_LT_UFO
+define p = g u c d s u~ c~ d~ s~ 
```

Comment:
please remove proton definition (I think is fine to keep jet definition in this case as is used also for the decay)

Reply:
Ok, changed it.